edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/WorkloadClientVersioned.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices.Edge.Util.Edged
     abstract class WorkloadClientVersioned
     {
         static readonly RetryStrategy TransientRetryStrategy =
-            new ExponentialBackoff(retryCount: 3, minBackoff: TimeSpan.FromSeconds(2), maxBackoff: TimeSpan.FromSeconds(30), deltaBackoff: TimeSpan.FromSeconds(3));
+            new ExponentialBackoff(retryCount: 2, minBackoff: TimeSpan.FromSeconds(1), maxBackoff: TimeSpan.FromSeconds(3), deltaBackoff: TimeSpan.FromSeconds(2));
```

Comment:
Why the change here?

Reply:
This is because, currently it is not possible to distinguish between iotedged operations failing because it cannot decrypt vs some other issue. So all operations are retried. 
This is to reduce the time taken for retrying all the decrypt operations, to prevent EdgeHub initialization from timing out.